geos.go

```diff
@@ -57,6 +57,30 @@ var (
 	errIndexOutOfRange     = Error("index out of range")
 )
 
+type PrecisionRule int
+
+// PrecisionRules.
+const (
+	// The output is always valid. Collapsed geometry elements
+	// (including both polygons and lines) are removed.
+	PrecisionRuleValidOutput PrecisionRule = 0 // C.GEOS_PREC_VALID_OUTPUT
```

Comment:
This should use `C.GEOS_PREC_VALID_OUTPUT`. If it's not defined in some versions of GEOS, then you will need to add it to `geos.h` inside a correct `#if`/`#endif` pair for the GEOS versions where it is not defined.

Reply:
See the contents of `geos.h` for how to define this conditionally. It can't be done in Go, and so needs to be done in C.